packages/resource-deployment/scripts/setup-cosmos-db.sh

```diff
@@ -85,8 +89,17 @@ onDemandScannerDbName="onDemandScanner"
 createCosmosDatabase "$scannerDbName"
 createCosmosDatabase "$onDemandScannerDbName"
 
-createCosmosCollection "a11yIssues" "$scannerDbName"
+# Increase throughput for below collection only in case of prod
 # Refer to https://docs.microsoft.com/en-us/azure/cosmos-db/time-to-live for item TTL scenarios
-createCosmosCollection "scanRuns" "$onDemandScannerDbName" "2592000"         # 30 days
-createCosmosCollection "scanBatchRequests" "$onDemandScannerDbName" "604800" # 7 days
-createCosmosCollection "scanRequests" "$onDemandScannerDbName" "604800"      # 7 days
+if [ $environment = "prod" ]; then
```

Comment:
Just to remove container/db name duplication can we move throughput to var and set in 'if' condition instead

Reply:
I have considered that option too, which says set the throughput to a default value and change if the default value only if env is prod. But are we ok to consider single throughput for all collections?